code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -293,6 +293,31 @@
 	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/exploration
 	heat_protection = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
 
+	//Cybersun Hardsuit
+	//A kind of side-grade to the explorer suit, sacrificing burn protection for brute. If you can kill the guy inside it, anyways.
+
+/obj/item/clothing/head/helmet/space/hardsuit/cybersun
+	name = "Cybersun hardsuit helmet"
+	desc = "A bulbous red helmet designed for scavenging in hazardous, low pressure environments. Has dual floodlights, and a 360 Degree view."
+	icon_state = "hardsuit0-cybersun"
+	item_state = "death_commando_mask"
+	item_color = "cybersun"
+	armor = list("melee" = 30, "bullet" = 35, "laser" = 15, "energy" = 15, "bomb" = 60, "bio" = 100, "rad" = 55, "fire" = 30, "acid" = 60, "stamina" = 15)
+	strip_delay = 600
+
+/obj/item/clothing/suit/space/hardsuit/cybersun
+	icon_state = "cybersun"
+	name = "Cybersun hardsuit"
+	desc = "A bulky, protective suit designed to protect against the perils facing Cybersun Employed Engineers, Researchers, and more as they head from the safety of \
+		more stable employment to the dangers of Nanotrasen Controlled Deep Space. Designed to get the job done despite on-site hazards in derelicts, laser armor was \
+		sacrificed in favor of more effective blunt armor plates and radiation shielding."
+	armor = list("melee" = 30, "bullet" = 35, "laser" = 15, "energy" = 15, "bomb" = 60, "bio" = 100, "rad" = 55, "fire" = 30, "acid" = 60, "stamina" = 15)
```

Comment:
Does this need an icon state?

Reply:
New hardsuit for the operative target, yeah. Otherwise it'd look like a stock engineering hardsuit, a design I'm not entirely against (As it'd actually be original and fit the other hardsuits' styles more (though it'd have to be pallet swapped and touched up a bit))

That all being said, in the time it's taken for this PR to settle down I've moved on from bee, so responses will be just a smidge slower. I still intend to see this PR to completion, though.

Reply:
oh I see the icon_state was at the top, I missed it